### DISCUSSION TITLE
No Emails from Yoda When Data is Added to Vault

### DISCUSSION THREAD

**Discussion by pradeep10kumar** [UtrechtUniversity/yoda#350] [Q&A] [answered]
No email notifications are sent by Yoda to the data manager when a data package is submitted into Vault. Is it possible to have this faetures added to YODA?

**Comment by lwesterhof** [comment]
Does the datamanager have mail notifications enabled? On every submit to vault the datamanager is notified and when enabled Yoda also sends an email notification.

A user can enable mail notifications under Settings in the user menu (click on username to open menu).

**Comment by Danny-dK** [comment, marked as answer]
If rodsadmin and in Yoda 1.8 using SURF hosted version, data managers that were also rodsadmin were not send an email (so that could also be a possibility). In 1.9 it is being sent at least for Yoda@WUR.

**Comment by Danny-dK** [comment]
@pradeep10kumar any update?

**Reply by pradeep10kumar** [reply]
@Danny-dK Thanks for the follow-up. It worked now for version 1.9.

**Reply by Danny-dK** [reply]
@pradeep10kumar  good to hear. Could you select your own response as an answer or any of the other responses? Then the discussion list makes it clear which things have no answers yet for the community to focus on. After selecting an answer you can close the discussion so that this discussion entry gets filtered to the closed section (doesn't get deleted; will still be available).

**Reply by Danny-dK** [reply]
@pradeep10kumar thanks for closing. In regards to choosing an answer, when you created a discussion you should be able to see mark as answer:

![image](https://github.com/UtrechtUniversity/yoda/assets/45395070/f1a3a114-5f81-4c31-951a-a3479b553426)

Which is a button available within a response. The thumbs up is not registered as an answer in git discussions.
